ocaml/xapi/xapi_vm_lifecycle.ml

```diff
@@ -352,7 +356,7 @@ let maybe_get_guest_metrics ~__context ~ref =
   then Some (Db.VM_guest_metrics.get_record_internal ~__context ~self:ref)
   else None
 
-(** Take an internal VM record and a proposed operation. Return None iff the operation
```

Comment:
"iff" means "if and only if" and comes from mathematical writing. I suspect it is not a typo.

Reply:
They probably meant "iff" indeed, but I'm not sure whether it is actually used correctly :)